site/zh-CN/userGuide/create_partition.md

```diff
@@ -4,17 +4,17 @@ related_key: Partition
 summary: Learn how to create a partition in Milvus.
 ---
 
-# Create a Partitions
+# 创建Partition
 
 {{fragments/translation_needed.md}}
 
-This topic describes how to create a partition in Milvus.
+本章描述如何在Milvus中创建分区(Partition)。
```

Comment:
In Milvus Tech Doc, we do not translate terms such as `collection`, `partition`, `field`, `schema`, etc. You can check [Milvus Glossary](https://milvus.io/docs/v2.0.x/glossary.md) to check what we do not translate. Please just leave them as the English words they were (this applies to all other cases in the below doc).